### PR TITLE
Add acceptance criteria, copy/paste button, and responsible roles into checklist results

### PR DESCRIPTION
**Pull request by tsuiui**

### CODE REVIEW

File: src/components/checklist-container.css [hunk 1]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
 }
 
-.checklist-item-title {
-  color: #000;
-  font-size: 1.8rem;
-  font-weight: bold;
-  margin-bottom: 24px;
-  display: inline-block;
+.checklist-item__roles {
+  display:block;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:155]:
```suggestion
  display: block;
```

File: src/components/checklist-container.css [hunk 2]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
+}
+
+.checklist-item__roles__job {
+  background-color: lightgrey;
+  border-radius:0.25rem;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:160]:
```suggestion
  border-radius: 0.25rem;
```

File: src/components/checklist-container.css [hunk 3]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
+  display:inline-block;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:161]:
```suggestion
  display: inline-block;
```

File: src/components/checklist-container.css [hunk 4]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
+  margin:0 0.5rem 1rem 0;
+  padding:0.5rem;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:163]:
```suggestion
  padding: 0.5rem;
```

File: src/components/checklist-container.css [hunk 5]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
-  display: inline-block;
+.checklist-item__roles {
+  display:block;
+}
+
+.checklist-item__roles__job {
+  background-color: lightgrey;
+  border-radius:0.25rem;
+  display:inline-block;
+  margin:0 0.5rem 1rem 0;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:162]:
```suggestion
  margin: 0 0.5rem 1rem 0;
```

File: src/components/checklist-container.css [hunk 6]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
+  padding:0.5rem;
+}
+
+.checklist-item__copy-button {
+  margin:0 0 2rem 0;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:167]:
```suggestion
  margin: 0 0 2rem 0;
```

File: src/components/checklist-container.css [hunk 7]
```diff
@@ -151,12 +151,20 @@ input[type="radio"]:checked + label {
...
-  font-size: 1.8rem;
-  font-weight: bold;
-  margin-bottom: 24px;
-  display: inline-block;
+.checklist-item__roles {
+  display:block;
+}
+
+.checklist-item__roles__job {
+  background-color: lightgrey;
```
Comment (m4thieulavoie) [inline src/components/checklist-container.css:159]: Could we not pull in design tokens instead?

**Review by m4thieulavoie — CHANGES_REQUESTED** [review]

File: src/components/checklist-container.css (hunk 7 above)
Comment (tsuiui) [reply src/components/checklist-container.css:159]: I have a feature/chameleon-migration branch started in my fork which was intended to be a UI rework to take the app to chameleon.  It makes more sense to me to do all the styling changes at once including using chameleon properly. Given how the existing styling works with classes I didn't want to spend a lot of time migrating the styles before the className fix on chameleon-web.

Comment (m4thieulavoie) [reply src/components/checklist-container.css:159]: I see what you mean and that's fair. However, design tokens are not in `chameleon-web` so we technically could use them here without affecting the class system of this repo 👍

**Review by m4thieulavoie — APPROVED** [review]

**Pull request merged**